python/fastquant/backtest/backtest.py

```diff
@@ -333,5 +358,7 @@ class CustomData(bt.feeds.PandasData):
                 sort_by=sort_by,
                 **optim_params
             )
-
-    return sorted_combined_df
+    if return_transactions:
+        return sorted_combined_df, history_dfs
```

Comment:
Let's merge `history_dfs` into one dataframe so that it's intuitive for users to access the transactions right away. No need to add the parameter columns, but let's add a new `strategy_id` column that is consistent of with the index of the corresponding strategy in `sorted_combined_df`.